core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -246,6 +247,14 @@ def install_routes
       end
     end
 
+    def install_solidus_starter_frontend
+      return unless install_solidus_starter_frontend?
+
+      say_status :install, "Solidus Starter Frontend"
+      ENV['LOCATION'] ||= 'https://raw.githubusercontent.com/nebulab/solidus_starter_frontend/master/template.rb'
```

Comment:
We can postpone merging this until SolidusStarterFrontend is moved from nebulab org to solidusio.